app/gui/language/ast-parser/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "ast-parser"
```

Comment:
I don't understand this name. This is not AST parser, right? it is a text parser, right? I think it should be named just "parser"